Connectors/Revit/Speckle.Connectors.RevitShared/HostApp/ElementUnpacker.cs

```diff
@@ -131,64 +132,37 @@ private List<Element> PackCurtainWallElementsAndStackedWalls(List<Element> eleme
       // If you wonder why revit is driving people to insanity, this is one of those moments.
       // See [CNX-851: Stacked Wall Duplicate Geometry or Materials not applied](https://linear.app/speckle/issue/CNX-851/stacked-wall-duplicate-geometry-or-materials-not-applied)
       || (element is Wall { IsStackedWallMember: true } wall && ids.Contains(wall.StackedWallOwnerId))
+      // Railings: Remove TopRail when parent railing is selected
+      // Prevents duplication since railing converter includes TopRail as a child element
+      // TODO: Consider adding HandRail support (also inherits from ContinuousRail)
+      || (
+        element is TopRail topRail
+        && doc.GetElement(topRail.HostRailingId) is Railing railing
+        && ids.Contains(railing.Id)
+      )
```

Comment:
with the extension method, we at least have consistent children collection between connector (`ElementUnpacker.GetElementsAndSubElementIdsFromAtomicObjects`) and converter (`RevitElementTopLevelConverterToSpeckle.GetElementChildren`). BUT this method is still pretty fragile (not happy). but ... scope creep 🤷🏼‍♂️